string model_path
string class_name_path
---
---
